charmander/tests/routes/example/test_crud.py

```diff
@@ -0,0 +1,126 @@
+"""
+Example CRUD routes tests.
+
+Tests are sunny day cases under the assumption that framework conventions
+handle most error conditions.
+
+"""
+from unittest.mock import patch
+
+from hamcrest import (
+    assert_that,
+    contains,
+    equal_to,
+    has_entries,
+    is_,
+)
+from microcosm_postgres.context import SessionContext, transaction
+from microcosm_postgres.identifiers import new_object_id
+from microcosm_postgres.operations import recreate_all
+
+from charmander.app import create_app
+from charmander.models.example_model import Example
+
+
+class TestExampleRoutes:
```

Comment:
Tests, basic but important. These tests are sample route tests with test the end-to-end flow of the routes. Our tests both locally and in CI use an actual postgres instance (or whatever persistence) so we can test any nuances of the persistence we might have. This makes reproducing and writing tests for weird persistence-related bugs possible.